[2023-06-14T23:09:23.397+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2023-06-14T23:09:19.244608+00:00 [queued]>
[2023-06-14T23:09:23.422+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2023-06-14T23:09:19.244608+00:00 [queued]>
[2023-06-14T23:09:23.423+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T23:09:23.424+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 2
[2023-06-14T23:09:23.425+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T23:09:23.463+0000] {taskinstance.py:1303} INFO - Executing <Task(BashOperator): download_dataset_task> on 2023-06-14 23:09:19.244608+00:00
[2023-06-14T23:09:23.477+0000] {standard_task_runner.py:55} INFO - Started process 2516 to run task
[2023-06-14T23:09:23.483+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'manual__2023-06-14T23:09:19.244608+00:00', '--job-id', '199', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpn76wk3g_']
[2023-06-14T23:09:23.484+0000] {standard_task_runner.py:83} INFO - Job 199: Subtask download_dataset_task
[2023-06-14T23:09:23.607+0000] {task_command.py:388} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2023-06-14T23:09:19.244608+00:00 [running]> on host 570ce802d94a
[2023-06-14T23:09:23.759+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-14T23:09:19.244608+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-14T23:09:19.244608+00:00
[2023-06-14T23:09:23.765+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-14T23:09:23.769+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'kaggle datasets download -d sherrytp/airline-delay-analysis -p /opt/***/airline-delay-analysis && unzip /opt/***/airline-delay-analysis/airline-delay-analysis.zip -d /opt/***/airline-delay-analysis']
[2023-06-14T23:09:23.806+0000] {subprocess.py:86} INFO - Output:
[2023-06-14T23:09:25.821+0000] {subprocess.py:93} INFO - airline-delay-analysis.zip: Skipping, found more recently modified local copy (use --force to force download)
[2023-06-14T23:09:26.097+0000] {subprocess.py:93} INFO - Archive:  /opt/***/airline-delay-analysis/airline-delay-analysis.zip
[2023-06-14T23:09:26.154+0000] {subprocess.py:93} INFO - replace /opt/***/airline-delay-analysis/airline delay analysis/20.csv? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
[2023-06-14T23:09:26.156+0000] {subprocess.py:93} INFO - (EOF or read error, treating as "[N]one" ...)
[2023-06-14T23:09:26.158+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-06-14T23:09:26.194+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-06-14T23:09:26.212+0000] {taskinstance.py:1326} INFO - Marking task as UP_FOR_RETRY. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20230614T230919, start_date=20230614T230923, end_date=20230614T230926
[2023-06-14T23:09:26.257+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 199 for task download_dataset_task (Bash command failed. The command returned a non-zero exit code 1.; 2516)
[2023-06-14T23:09:26.310+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-06-14T23:09:26.375+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
